agents/supervisor/supervisor.go

```diff
@@ -496,12 +496,15 @@ func (s *Supervisor) processParams(agentID string, agentProcess *agentpb.SetStat
 	}
 	switch agentProcess.Type {
 	case inventorypb.AgentType_NODE_EXPORTER:
+		templateParams["paths_base"] = s.paths.PathsBase
```

Comment:
I don't want to be picky, but, shouldn't it be `base_path`?
@PaulJacobs-percona help us, please.

Reply:
@percona-csalguero This name comes from ticket 
https://jira.percona.com/browse/PMM-5669
Probably because we have also exporter_base, thats why is base second here.

Reply:
Looks like an externally-visible param name was chosen to correspond with an internal param. `base_path` is more intuitive than `paths_base` but I would imagine documentation would explain what the param does anyway.